eks-distro-base/minimal-image-rebuilder.sh

```diff
@@ -0,0 +1,12 @@
+set -e
```

Comment:
add the bash shebang and copyright disclaimer? See https://github.com/aws/eks-distro-build-tooling/blob/main/pr-scripts/create_pr.sh#L1

Reply:
Done.